fireant/dataset/data_blending.py

```diff
@@ -77,9 +77,10 @@ def __init__(self, primary_dataset, secondary_dataset, dimension_map):
         self.fields = DataSet.Fields(
             ordered_distinct_list_by_attr(
                 [
-                    *_wrap_dataset_fields(primary_dataset),
                     *_wrap_dataset_fields(secondary_dataset),
-                ]
+                    *_wrap_dataset_fields(primary_dataset)
+                ],
+                keep_last=True  # This makes sure we keep the blended field if there are fields with the same name
```

Comment:
Given the first fields in the list are from the secondary dataset and the later fields are the primary dataset, wouldn't keep_last=true make us keep the primary?

Reply:
The reason why this works is a complex side effect of how we build up multiple blended datasets in dashmore. At best this is a temporary fix (that does work for all our cases) until we have properly come up with a solution for all duplicated alias problems.